src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1319,7 +1319,8 @@
           "description": "A number representing the priority or importance of the result.",
           "type":"number",
           "minimum": 0.0,
-          "maximum": 100.0
+          "maximum": 100.0,
+          "default": 0.0
```

Comment:
>0.0 [](start = 21, length = 3)

i thought you argued for -1 as a default?

Reply:
Yes, but that hasn't been approved by the TC yet. As the spec stands, it's 0. It's on the agenda to change it.

---
In reply to: [244648426](https://github.com/Microsoft/sarif-sdk/pull/1193#discussion_r244648426) [](ancestors = 244648426)